### ISSUE TITLE
Player bugs

### ISSUE THREAD

**Issue by JrTheDragon01** [Moontemple/Asteroid#21]
```Exception in thread Thread-1:
Traceback (most recent call last):
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 38, in run
    msg = self.queue.get(timeout=0.1)
  File "/usr/lib/python3.5/queue.py", line 172, in get
    raise Empty
queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 45, in run
    self.play()
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 96, in play
    path = self.get_path_from_database()
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 83, in get_path_from_database
    song = MusicDB(os.environ["MUSIC_DB_PATH"]).get_by_rowid(n_item[0]+1)[0]
IndexError: tuple index out of range```

**Comment by JrTheDragon01** [comment]
Occurred again:
```Exception in thread Thread-1:
Traceback (most recent call last):
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 38, in run
    msg = self.queue.get(timeout=0.1)
  File "/usr/lib/python3.5/queue.py", line 172, in get
    raise Empty
queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 45, in run
    self.play()
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 96, in play
    path = self.get_path_from_database()
  File "/home/pi/Documents/Asteroid/src/main/python/player/AudioHandler.py", line 83, in get_path_from_database
    song = MusicDB(os.environ["MUSIC_DB_PATH"]).get_by_rowid(n_item[0]+1)[0]
IndexError: tuple index out of range```

**Comment by JrTheDragon01** [comment]
For songs of equal rating, player plays song most recently added rather than song added earliest

**Comment by JrTheDragon01** [comment]
> Occurred again; error code was same.

and again

**Comment by Moontemple** [comment]
@JrTheDragon01 and I believe this is part of the larger issue relating to SQL syntax going sanitised (fixed in e0ac1964303b6f692c8e6f06b1b512ae8fc195ea for database building), however it highlights that the audioplayer doesn't validate song id before attempting to fetch data - going to keep the issue open until that's fixed.

**Comment by Moontemple** [comment]
Fixed in abbc07fcb202bd993ccf47147fe954f57fac47bd !
